Add missing information to readme.md (#2)
magma/tsc: README.MD

# DRAFT

> 

Reviewer: Obv will want to remove this, just a reminder

### Mailing List

> The TSC for Magma Core Foundation can be reached at their [mailing list](https://lists.magmacore.org/g/magma-tsc)

Reviewer: ```suggestion
The TSC for Magma Core Foundation can be reached at their [mailing list](https://lists.magmacore.org/g/magma-tsc).
```

### Slack

> The Magma Core Foundation maintains a [Slack Workspace](https://join.slack.com/t/magmacore/shared_invite/zt-g76zkofr-g6~jYiS3KRzC9qhAISUC2A) for communication and collaboration. The Magma Core Foundation [Slack Workspace](https://join.slack.com/t/magmacore/shared_invite/zt-g76zkofr-g6~jYiS3KRzC9qhAISUC2A) is open for anyone to join.

Reviewer: Only add the hyperlink once

### Calendars and Meetings

> The Magma Core Foundation maintains a [public calendar](https://calendar.google.com/calendar/embed?src=c_gbiu1t7a67ika1th2smldeh19s%40group.calendar.google.com&ctz=America%2FChicago) for TSC and community meetings. These meetings are open for anyone to join.

Reviewer: ```suggestion
The Magma Core Foundation maintains a [public calendar](http://bit.ly/MagmaPublicCalendar) for TSC and community meetings. These meetings are open for anyone to join.
```

### CLA

Everyone must sign the CLA prior to making a contribution. You may either sign the CLA on your own, or your company can sign it for you. At a high level, if you aren't covered by a CLA, you will be notified the first time you open a PR. Please see our [documentation on the process](./CLA.md).

## Members

> The voting members of the Magma Core Foundation TSC are listed [here](https://github.com/magma/Governance/wiki/Magma-TSC-Members)

Reviewer: Here and throughout the changes: use descriptive hyperlink text, i.e. not "here" https://magma.github.io/magma/docs/next/contributing/contribute_conventions#all-documentation

## Policies and procedures

The Magma Core Foundation TSC is governed by the Technical Charter. The Charter provides a foundational structure for the TSC on topics such as its scope, how to make decisions, and how to make changes to itself. At the same time, it grants the TSC a high degree of freedom when determining how to implement the policies of the Magma Core Foundation.

The following policies and procedures have been adopted by the TSC.

### Making decisions

A majority of the eligible voting members participating at a meeting, or participating in an email vote separate from a meeting, shall constitute a quorum. A majority of the eligible voting members participating when a quorum is present shall be required to approve other actions of the Committee.

### Merging PRs into the TSC repository

Pull requests that do not change the charter or governance of the TSC can be merged into this repository provided the following conditions have been met:

> *There are no outstanding objections
> *There are two approvals by TSC members
> *The PR has been open for at least 72 hours
> *Pull requests that change governance of the TSC (excluding the charter) must be open for at least 14 days, unless consensus is reached in a meeting with quorum of voting members.

Reviewer: Use `- ` instead of `*` for lists
```suggestion
- There are no outstanding objections
- There are two approvals by TSC members
- The PR has been open for at least 72 hours
- Pull requests that change governance of the TSC (excluding the charter) must be open for at least 14 days, unless consensus is reached in a meeting with quorum of voting members.
```

Pull requests that change the charter of the TSC must meet any requirements in the charter.

If consensus cannot be reached, a pull request may still be landed after a vote by the Voting members to override outstanding objections.

### Fast-Tracking PRs

Special exception is made for pull requests seeking to make any of the following changes to this repository:

Errata fixes.
Editorial changes.
Meeting minutes.
Updates to team lists.
Doc fixes.

> Charter changes cannot be fast-tracked.

Reviewer: Make this a list, like above, by prepending `- ` to each line. Also remove the periods from each line
